media/css/firefox/whatsnew/whatsnew-88-en.scss

```diff
@@ -0,0 +1,150 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/notification-bar';
+@import 'includes/header';
+
+
+//* -------------------------------------------------------------------------- */
+// Main content
+.wnp-content-main {
+    padding: 0;
+    text-align: center;
+
+    @media #{$mq-tall} {
+        padding-top: $layout-md;
+        margin-bottom: $layout-lg;
+    }
+}
+
+.wnp-main-image {
+    margin: 0 auto $layout-sm;
+}
+
+.c-animated-graphic {
+    width: 100%;
+    max-width: 450px;
+    display: block;
+    margin: 0 auto $layout-sm;
+
+    &.wnp-vid-1 {
+        max-width: 620px;
+    }
```

Comment:
This class is for the first variation. It's image/video is larger than the default `450px`.